app/screens/AddChildSetup.tsx

```diff
@@ -0,0 +1,573 @@
+};
+const styles = StyleSheet.create({
+  containerView: {
+    backgroundColor: bgcolorWhite2,
+    flex: 1
+  },
+  flex2Style: {
+    alignItems: 'flex-start'
+  },
+  flexRow1: {
+    marginTop: 10
+  },
+  scrollViewStyle: {
+    padding: 0,
+    paddingTop: 0
+  },
+  textInputStyle: {
+    width: '100%',
+  },
+  textParentInfoStyle: {
+    textAlign: 'center'
+  },
+  dividerStyle: {
+    marginEnd: 20
+  },
+  orDividerStyle: {
+    width: 172,
+    alignSelf: 'center',
+  },
+  uploadTextStyle: {
+    color: "#1CABE2"
+  },
+})
+const AddChildSetup = ({ route, navigation }: Props): any => {
+  const { t } = useTranslation();
+  const [relationship, setRelationship] = useState('');
+  const [userRelationToParent, setUserRelationToParent] = useState();
+  const [relationshipname, setRelationshipName] = useState('');
+  const [birthDate, setBirthDate] = useState<Date>(new Date());
+  const [plannedTermDate, setPlannedTermDate] = useState<Date>();
+  const [isImportRunning, setIsImportRunning] = useState(false);
+  const [isPremature, setIsPremature] = useState<string>('false');
+  const [isExpected, setIsExpected] = useState<string>('false');
+  const [name, setName] = React.useState('');
+  const [loading, setLoading] = useState(false);
+  const [isImportAlertVisible, setImportAlertVisible] = useState(false);
+  const actionSheetRefImport = createRef<any>();
+  const netInfo = useNetInfoHook();
+  let relationshipData = useAppSelector(
+    (state: any) =>
+      state.utilsData.taxonomy.allTaxonomyData != '' ? JSON.parse(state.utilsData.taxonomy.allTaxonomyData).parent_gender : [],
+  );
+  const relationshipToParent = useAppSelector(
+    (state: any) =>
+      state.utilsData.taxonomy.allTaxonomyData != '' ? JSON.parse(state.utilsData.taxonomy.allTaxonomyData).relationship_to_parent : [],
+  );
+  const languageCode = useAppSelector(
+    (state: any) => state.selectedCountry.languageCode,
+  );
+  const childAge = useAppSelector(
+    (state: any) =>
+      state.utilsData.taxonomy.allTaxonomyData != '' ? JSON.parse(state.utilsData.taxonomy.allTaxonomyData).child_age : [],
+  );
+  const childList = useAppSelector(
+    (state: any) => state.childData.childDataSet.allChild != '' ? JSON.parse(state.childData.childDataSet.allChild) : [],
+  );
+  const actionSheetRef = createRef<any>();
+  const [gender, setGender] = React.useState(0);
+  const dispatch = useAppDispatch();
+  const sendData = (data: any): any => { // the callback. Use a better name
+    setBirthDate(data.birthDate);
+    setPlannedTermDate(data.plannedTermDate);
+    const myString = String(data.isPremature);
+    setIsPremature(myString);
+    setIsExpected(String(data.isExpected));
+  };
+  let genders = useAppSelector(
+    (state: any) =>
+      state.utilsData.taxonomy.allTaxonomyData != '' ? JSON.parse(state.utilsData.taxonomy.allTaxonomyData).child_gender : [],
+  );
+
+  genders = genders.map((v: any) => ({ ...v, title: v.name })).filter(function (e: any) {

```

Comment:
What is v and e? 
Can we use more meaningful names?

Reply:
Above code is used for genders and it is old code. here v is used for items in genders array and e is items used for filter data through gender.